README.md

```diff
@@ -1,11 +1,14 @@
 # A-Frame Core
 
-![travis](https://travis-ci.org/aframevr/aframe-core.svg)
-
-> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
+[![build status][travis-image]][travis-url]
+[![NPM version][npm-image]][npm-url]
+[![Downloads][downloads-image]][downloads-url]
+[![js-semistandard-style][semistandard-image]][semistandard-url]
 
 A-Frame is a library for creating virtual reality web experiences.
 
+> There are currently plans for this repository to be the canonical [`aframe`](https://github.com/aframevr/aframe).
```

Comment:
nitpick but can we move this above the tagline? i feel it interrupts the introduction.